README.md

```diff
@@ -32,7 +32,8 @@ Tested Development environment: [Slicer Windows Dev Environment].
 
 1. Run CMake (cmake-gui) from the Windows Start Menu
     * Set the build directory `C:\slicerSALT-Debug` or `C:\slicerSALT-Release`
-    * Select your compiler: Visual Studio 12 2013 Win64
+    * Select your compiler: Visual Studio 14 2015 Win64
+    * If using a newer version of Visual Studio, be sure to specify the 2015 toolset (v140)
```

Comment:
An other update to do: Instead of `QT_QMAKE_EXECUTABLE`, `Qt5_DIR` should be set.

Reply:
Yep.  I plan to do a full pass on the build instructions to bring them up to date.